### ISSUE TITLE
Guild Bank don't obey Sorting > When

### ISSUE THREAD

**Issue by esvarc** [arkayenro/arkinventory#805]
Actually I do have one sorting method and all windows are set in Window > Style > Sorting > When "On Window Open". 

All windows except Guild Bank really sort on open or when I press manually "Sort" button. Guild Bank behave like it has "Instantly" option. BTW even when I set GB to "Manually" it sorts everytime I put or remove item from GB.

It of coarse produce big lag spikes as all items in GB are refreshed. This behave I see in actual latest release version (3.06.17) and in latest beta build (3.07.00_BETA_3c).

**Comment by arkayenro** [comment]
i think theres an issue where the guild bank keeps using the defaults and not what you set, just need to find where it is

**Comment by esvarc** [comment]
Sort method is in all settings same. Same sort method with "On Window Open" in all styles including default. Only guild bank won't obey it. I will test it when I delete arkinventory saved variebales completely, to see what happens.

**Comment by esvarc** [comment]
So yes it is that way. Here are steps to reproduce bug:

 
1. I did delete file ArkInventory.lua  in SavedVariables directory
2. Created user defined sort method "ascending" > item type and subtype > item rarity > item (stat) level > item name
3. Assigned taht sort method in style under Sort Method, "When" is "On Window Open"
4. Open guild bank and remove item, it get instantly sorted instead leaving slot without item.

I'm attaching stored variables if you want simulate or check it I did must change extension to TXT to be able upload it here. 

[ArkInventory.lua.txt](https://github.com/arkayenro/arkinventory/files/390595/ArkInventory.lua.txt)
